Jellyfin.Plugin.Webhook/Templates/Ntfy.handlebars

```diff
@@ -1,47 +1,110 @@
+##REMOVE THESE LINES!!! ----->
+##This JSON template uses markdown (e.g. "---", "**" & "## ") which is supported by ntfy.sh on browser. Don't forget to add following Request Header:
+##Key: X-Markdown
+##Value: true
+##
+##To publish as JSON, use the ntfy root URL, not the topic URL:
+##Correct: https://ntfy.sh[:PORT]
+##Incorrect: https://ntfy.sh[:PORT]/jellyfin
+##
+##The topic ("jellyfin"), priorities (1-5) and of course titles & messages are specified within this JSON template. Modify them as you wish.
+##<----- REMOVE THESE LINES!!!
 {
     "topic": "jellyfin",
-    "tags": ["octopus"],
     {{#if_equals NotificationType 'PlaybackStart'}}
-        "title": "{{{NotificationUsername}}} Playback Started",
-        "priority": 3,
-        "attach": "{{{ServerUrl}}}/Items/{{{ItemId}}}/Images/Primary",
+        "priority": 2,
+        "tags": ["arrow_forward"],
+        "attach": "{{{ServerUrl}}}Items/{{{ItemId}}}/Images/Primary",
+        "actions": [{ "action": "view", "label": "Visit Jellyfin", "url": "{{{ServerUrl}}}web/index.html#!/details?id={{{ItemId}}}" }],
```

Comment:
For 10.9 the route to the item changed to `{{{ServerUrl}}}/web/#/details?id={{ItemId}}`

Reply:
thanks for the hint.
I changed it, although it still worked on my Jellyfin instance :)
`{{{ServerUrl}}}web/#/details?id={{ItemId}}`
I had to remove the first slash though.